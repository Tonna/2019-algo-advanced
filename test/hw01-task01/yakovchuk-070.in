15
1 1 2 3 3 1 2 1 2 9 3 11 10 6
7
4 7
11 2
9 6
13 5
7 8
14 2
2 1
