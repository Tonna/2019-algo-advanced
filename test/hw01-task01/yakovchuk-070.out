12
10
17
24
18
20
2
